src/frontend/components/app/sidebar/sidebar-branding.tsx

```diff
@@ -55,7 +55,10 @@ const SidebarBranding: React.FC<Props> = (props) => {
         />
       ) : <h1>{companyName}</h1>}
     </StyledLogo>
-  )
+    { branding.additionalText &&
+      <span>{branding.additionalText}</span>
```

Comment:
I'd use `Text` (`import { Text } from '@adminjs/design-system`). This uses AdminJS global styles.